.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v5.0.0
+  hooks:
+  - id: check-added-large-files
+  - id: check-case-conflict
+  - id: check-toml
+  - id: check-yaml
+  - id: debug-statements
+  - id: end-of-file-fixer
+  - id: forbid-new-submodules
+  - id: trailing-whitespace
+
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.8.2
+  hooks:
+    # - id: ruff
+    #   args: ["--fix", "--show-fixes"]
+    - id: ruff-format
+      exclude: notebooks/techniques/evaluating_information_retrival_techniques_mongondb_langchain.ipynb
```

Comment:
Curious  was this notebook excluded?

Reply:
The notebook failed to parse, but I couldn't track down why.  😅